components/server/src/container-module.ts

```diff
@@ -270,6 +270,14 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
         return createClient(BillingServiceDefinition, createChannel(config.usageServiceAddr));
     });
 
+    bind<IDEServiceClient>(IDEServiceDefinition.name).toDynamicValue((ctx) => {
```

Comment:
@iQQBot could that be bound in singleton scope as well?

Asking, because of investigation of memory leak issue introduced very recently.

Reply:
@iQQBot, I thought so ;-) 
see https://github.com/gitpod-io/gitpod/pull/13456

Reply:
@iQQBot, would you mind adjusting in same way?